src/main/java/com/puppycrawl/tools/checkstyle/ConfigurationLoader.java

```diff
@@ -471,7 +473,7 @@ public static Configuration loadConfiguration(InputSource configSource,
      * @throws CheckstyleException if the string contains an opening
      *                           {@code ${} without a closing
      *                           {@code }}
-     * @noinspection MethodWithMultipleReturnPoints
+     * @noinspection MethodWithMultipleReturnPoints, MethodOnlyUsedFromInnerClass
```

Comment:
please place reason of suppress to javadoc description, sad that I already missed this for another suppression.

Reply:
Done.